src/app/features/auth/components/Card/card.tsx

```diff
@@ -0,0 +1,52 @@
+import Image, { StaticImageData } from "next/image";
+import { Button } from "../button";
+import pax from "@/assets/images/home/pax.png";
+import tv from "@/assets/images/home/tv.png";
+import wifi from "@/assets/images/home/wifi.png";
+import bed from "@/assets/images/home/bed.png";
+
+interface CardProps {
+  image: StaticImageData;
+  name: string;
+  price: string;
+}
+
+const Card: React.FC<CardProps> = ({ image, name, price }) => {
+  return (
+    <div className="}] mr-10 h-[570px] w-[550px] rounded-2xl bg-[rgb(254,255,243)] shadow-xl">
```

Comment:
try using `flexbox` when styling this component para di hard coded yung margins at maging easier satin pag nagapply ng responsiveness sa pages at components